src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -150,8 +148,10 @@ void CascadiaSettings::_CreateDefaultProfiles()
     powershellProfile.SetUseAcrylic(false);
     powershellProfile.SetName(L"PowerShell");
 
-    _profiles.emplace_back(defaultProfile);
+    _profiles.emplace_back(cmdProfile);

```

Comment:
Do these need to be inserted in the other order? That is, will this leave cmd.exe as the profile on ctrl+shift+1?

If PowerShell is the default, it probably belongs on Ctrl+Shift+1 by default as well (by nature of the ordering).

Reply:
Thanks for pointing out, I have now updated this.